src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbLogger.java

```diff
@@ -210,15 +212,31 @@ private String scanRequestToString(ScanRequest scanRequest) {
             + ", values=" + scanRequest.getExpressionAttributeValues();
     }
 
+    private String putItemRequestToString(PutItemRequest putRequest) {
+        return (!isNullOrEmpty(putRequest.getConditionExpression())
```

Comment:
why not use a joiner and don't return it with the opening "," (have callers handle that)?

Reply:
fixed in 6fb6604ff5996f2b174b052fb1b38d45fbc9ab63